tests/main/test_main_general.py

```diff
@@ -71,6 +71,19 @@ def test_show_help_when_no_input(mocker: MockerFixture) -> None:
     assert print_help_mock.called
 
 
+def test_no_args_has_default(monkeypatch: pytest.MonkeyPatch) -> None:
```

Comment:
I don't think this is the right test. We want to see that the `pyprojct.toml` values are respected, not that no values have default values.

Reply:
I know but... Do you have any idea how to test that for *all* values without setting them by hand? I can't find an easy way to test the code in `__main__.py` easily.

https://github.com/koxudaxi/datamodel-code-generator/blob/ef0c97cb34d753aa831fe9f4e71df399d90d746b/src/datamodel_code_generator/__main__.py#L366-L377

Reply:
CC: @koxudaxi